nodes/generators_extended/receive_from_sorcar.py

```diff
@@ -0,0 +1,86 @@
+import bpy
+import ast
+from bpy.props import StringProperty
+
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode
+from sverchok.utils.modules.geom_sorcar import make_sorcar_logo
+
+class SvReceiveFromSorcarNode(bpy.types.Node, SverchCustomTreeNode):
+    ''' Receive Mesh Data From Sorcar '''
```

Comment:
we have a useful scheme for this , and it makes tooltips for out menus automatically

```
    """
    Triggers: sorcar sc receive
    Tooltip: Receive Mesh Data From Sorcar
    
    A short description for reader of node code <---not mandatory, but you could include links of interest.
    """
```

Reply:
Thanks!
I don't have any links of interest (maybe except for this PR as it has some info on the top), so I'll create a separate small description and add some comments in the code.